meteor/client/ui/i18n.ts

```diff
@@ -45,7 +45,7 @@ const i18nOptions = {
 	},
 
 	backend: {
-		loadPath: '/locales/{{lng}}/{{ns}}.json',
+		loadPath: './locales/{{lng}}/{{ns}}.json',
```

Comment:
```suggestion
		loadPath: Meteor.absoluteUrl('./locales/{{lng}}/{{ns}}.json'),
```
Same as above, I expect this needs to be "absolutified" by hand, since I don't think Meteor will have a way to baseUrl this.